libs/s25main/Loader.cpp

```diff
@@ -604,15 +604,8 @@ void Loader::fillCaches()
                     }
                     const libsiedler2::ImgDir imgDir = toImgDir(dir);
 
-                    unsigned overlayIdx = bob_jobs->getOverlayIdx(id, fat, imgDir, ani_step);
-                    if(overlayIdx == 188 && fat)
-                    {
-                        // No fat version(?)
-                        overlayIdx = bob_jobs->getOverlayIdx(id, false, imgDir, ani_step);
-                        fat = false;
-                    }
                     bmp.add(dynamic_cast<glArchivItem_Bitmap_Player*>(bob_jobs->getBody(fat, imgDir, ani_step)));
```

Comment:
was this a bug with non-gold-edition files? (i.e 1.55+mission-cd) - anyhow we could fix it with an override file then :-D

Reply:
No idea that's why I asked. With gold edition it works fine so no problem here ;)

Reply:
perhaps it was some different bug with animation system or so, which one worked around a while ago.